snake.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="snake.css">
+  <title>Document</title>
+</head>
+<body>
+    <div class="row">
+      <div class="row-1">3 colunms</div>
+      <div class="row-2">2 colunms</div>
+      <div class="row-3">1 colunm</div>
+    </div>
+    <div class="middle">
+      <div class="middle-1">
+        <div class="middletop">
+          <div class="square square-1">1</div>
+          <div class="square square-2">2</div>
+          <div class="square square-3">3</div>
+        </div>
+        <div class="middlebottom">
+          <div class="square square-4">4</div>
+          <div class="square square-5">5</div>
+          <div class="square square-6">6</div>
+        </div>
+      </div>
+      <div class="middle-2">
+          <div class="left">
+            <div class="leftcol square-1">1</div>
+            <div class="leftcol square-2">2</div>
+            <div class="leftcol square-3">3</div>
+          </div>
+          <div class="right">
+            <div class="rightcol square-4">4</div>
+            <div class="rightcol square-5">5</div>
+            <div class="rightcol square-6">6</div>
+          </div>
+      </div>
+      <div class="middle-3">
+        <div class="square square-1">1</div>
+        <div class="square square-2">2</div>
+        <div class="square square-3">3</div>
+        <div class="square square-4">4</div>
+        <div class="square square-5">5</div>
+        <div class="square square-6">6</div>
+      </div>
+    </div6>
```

Comment:
```suggestion
      <div class="container">
        <div class="square square-1">1</div>
        <div class="square square-2">2</div>
        <div class="square square-3">3</div>
        <div class="square square-4">4</div>
        <div class="square square-5">5</div>
        <div class="square square-6">6</div>
      </div>
```
залиш тільки це
все інше зайве і зміни стилі щоб все працювало. це завдання про адаптивний дизайн, коли в тебе верстка змінюється, а не просто щось ховати, а щось показувати